test/e2e/framework/sessionconfig.go

```diff
@@ -237,18 +238,29 @@ func (cfg SessionConfig) DeleteFARs() []*ie.IE {
 	}
 }
 
-func (cfg SessionConfig) SessionIEs() []*ie.IE {
+func (cfg SessionConfig) CreateURR (id uint32, triggers uint16) *ie.IE {
+	urr := ie.NewCreateURR(
+		ie.NewURRID(id),
+		ie.NewMeasurementMethod(0, 1, 1), // VOLUM=1 DURAT=1
+		ie.NewReportingTriggers(triggers))
+	// for Quota Validity Time report trigger is 15th bit in host order
+	if (triggers & 0x80) != 0 {
+		urr.Add(ie.NewQuotaValidityTime(time.Now()))
+
+		valTimer, _ := urr.FindByType(ie.QuotaValidityTime)
+		for i, _ := range valTimer.Payload {
+			valTimer.Payload[i] = 0
+		}
+		valTimer.Payload[3] = 10
+
+	}
+	return urr
+}
+
+func (cfg SessionConfig) SessionIEs(triggers uint16) []*ie.IE {
```

Comment:
It would be better to add `ReportingTriggers` field in `SessionConfig` (note that zero value is the default so there will be no need to set it). This structure is also used by `pfcpstub` in FPP test setup so it will have to be changed if the argument is added here.

Reply:
Thanks for the note. Changed as a type field of `SessionConfig`